camayoc/qcs_models.py

```diff
@@ -330,9 +331,21 @@ def equivalent(self, other):
                         default_port)):
                     return False
             if key == 'options':
+                if self.source_type in QCS_HOST_MANAGER_TYPES:
+                    if hasattr(self, 'options'):
+                        ssl_verify = self.options.get('ssl_cert_verify', True)
+                    else:
+                        ssl_verify = True
+                    if other.get(key).get('ssl_cert_verify') != ssl_verify:
+                        return False
                 if self.source_type == 'satellite':
-                    if other.get(key).get('satellite_version') != '6.2':
+                    if hasattr(self, 'options'):
+                        sat_ver = self.options.get('satellite_version', '6.2')
+                    else:
+                        sat_ver = '6.2'
+                    if other.get(key).get('satellite_version') != sat_ver:
```

Comment:
This logic is awkward and I'm interested to know if you have suggestion for some other flow.

Reply:
I don't have any good suggestion right now. The only thing I see is that if `other` does not have `key` then it will return None and then will raise AttributeError since None object does not have the `get` method. I would recommend calling `other.get(key, {}).get('satellite_version')`.